data-assets/acm_right_sizing_grafana_dashboard.yaml

```diff
@@ -1663,16 +1667,51 @@ data:
             "queryValue": "",
             "skipUrlSync": false,
             "type": "custom"
+          },
+          {
+            "allValue": null,
+            "current": {
+              "selected": false,
+              "text": "5",
+              "value": "5"
+            },
+            "description": null,
+            "error": null,
+            "hide": 0,
+            "includeAll": false,
+            "label": null,
+            "multi": false,
+            "name": "top_n",
+            "options": [
+              {
+                "selected": true,
+                "text": "5",
+                "value": "5"
+              },
+              {
+                "selected": false,
+                "text": "10",
+                "value": "10"
+              },
+              {
+                "selected": false,
+                "text": "20",
+                "value": "20"
+              }
+            ],
+            "query": "5, 10, 20",
+            "skipUrlSync": false,
+            "type": "custom"
           }
         ]
       },
       "time": {
-        "from": "now-7d",
+        "from": "now-90d",
```

Comment:
Is this graph data range ? If yes lets change it to either 7 days or max 30 days.

Reply:
Done.

Reply:
This will effect the entire dashboard (e.g. the recommendations will be based on the last 7 days)

Reply:
When you say recommendations will be based on the last 7 days : Don't we give recommendations based on aggregate days rather than this range ? I was under impression that we time range will be utilized just to showcase the graph data while aggregation days will be utilized to show table with recommendations.

Reply:
> This will effect the entire dashboard (e.g. the recommendations will be based on the last 7 days)

@Anxhela21  Don't we give recommendations based on aggregate days rather than this range ? I was under impression that we time range will be utilized just to showcase the graph data while aggregation days will be utilized to show table with recommendations.

Reply:
discussed in scrum